tests/integration/test_logcollector/test_configuration/test_basic_configuration_target.py

```diff
@@ -0,0 +1,168 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: The 'wazuh-logcollector' daemon monitors configured files and commands for new log messages.
+       Specifically, these tests will check if the logcollector detects that a custom socket is
+       undefined if the 'target' attribute of the 'out_format' tag has the name of an unexistent
+       socket (invalid value). Log data collection is the real-time process of making sense out
+       of the records generated by servers or devices. This component can receive logs through
+       text files or Windows event logs. It can also directly receive logs via remote syslog
+       which is useful for firewalls and other such devices.
+
+components:
+    - logcollector
+
+suite: configuration
+
+targets:
+    - agent
+    - manager
+
+daemons:
+    - wazuh-logcollector
+
+os_platform:
+    - linux
+    - macos
+    - solaris
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Solaris 10
+    - Solaris 11
+    - macOS Catalina
+    - macOS Server
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/log-data-collection/index.html
+    - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html#out-format
+
+tags:
+    - logcollector_configuration
+'''
+
+import pytest, sys
+
+from pathlib import Path
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.constants.platforms import WINDOWS
+from wazuh_testing.constants.daemons import LOGCOLLECTOR_DAEMON
+from wazuh_testing.modules.logcollector import configuration as logcollector_configuration
+from wazuh_testing.modules.logcollector import patterns
+from wazuh_testing.modules.logcollector import utils
+from wazuh_testing.tools.monitors import file_monitor
+from wazuh_testing.utils import callbacks, configuration
+from wazuh_testing.utils.services import control_service
+from wazuh_testing.utils.file import truncate_file
+
+from . import TEST_CASES_PATH, CONFIGURATIONS_PATH
+
+
+# Marks
+pytestmark = [pytest.mark.linux, pytest.mark.darwin, pytest.mark.sunos5, pytest.mark.tier(level=0)]
+
+# Configuration
+no_restart_windows_after_configuration_set = True
```

Comment:
Is this necessary if this test doesn't run on Windows?

Reply:
Removed!